micrometer-registry-newrelic-telemetry/runtime/src/main/java/io/quarkiverse/micrometer/registry/newrelic/telemetry/newrelic/NewRelicRegistryConfigAlternative.java

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
As a general rule, we don't use copyrights in files..

Reply:
Ok, will kick it out of my header config and fix these.